docker/experimental/Dockerfile

```diff
@@ -74,7 +74,7 @@ COPY third_party/ third_party/
 COPY build_torch_xla_libs.sh .
 
 # TODO: Remove this when it's not required anymore
-ENV XLA_SANDBOX_BUILD=0
+ENV XLA_SANDBOX_BUILD=1
```

Comment:
Do we have plan to remove sandbox in the future?

Reply:
I'll defer to @stgpetrovic on that. My understanding is that we need sandboxing to take advantage of remote build execution.

Reply:
The sandbox is required by some bug; bazel complains about some llvm mlir deps, that's why the sandbox was introduced before this migration. Once we're running remote, there's no need for sandbox,

Reply:
OK this makes sense: this build was using 8 jobs and was running locally before. We can either leave it at that, or make the RAM a bit bigger and make it much faster.

For now I'd like to fix this so I'll make it run as it did before; with the new .bazelrc we default to shm/sandboxed.